manage/installing/installation_windows10_wsl.rst

```diff
@@ -0,0 +1,271 @@
+
+The following tutorial will lead you trought all the steps to install and use Plone CLI on your Windows device.
+
+Ubuntu
+======
+
+Pre Settings
+------------
+
+Before installing the shell to later configure in, the first thing to do is to enable the optional feature for a subsystem on windows.
+Therefor, the programm Windows PowerShell has to be launched (as administrator! - right-click the application and choose "launch as admin").
+Do not launch the Powershell ISE application, because it won't work.
+In the shell the following command has to be entered.
+
+.. code-block :: powershell
+
+    Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+
+If necessary, the device has to be restarted afterwards.
+
+Download And Installing Ubuntu
+------------------------------
+
+After giving free the feature for a subsystem, the distribution package of Ubuntu can be downloaded using the following command.
+
+.. code-block :: powershell
+
+    Invoke-WebRequest -Uri https://aka.ms/wsl-ubuntu-1804 -OutFile Ubuntu.appx -UseBasicParsing
+
+To install the downloaded package, execute the following command.
+
+.. code-block :: powershell
+
+    Add-AppxPackage .\Ubuntu.appx
+
+Setting Up Ubuntu
+-----------------
+
+After installing the application, it can be found in the startmenu or in the apps, or can be found by searching it.
+After starting up Ubuntu, it take some time to process futher installation steps and establish the needed components.
+An username and password is required, indepentend of the Windows user data.
+
+Because most distributions just deliver an outdated catalog within the download, it is recommended to instantly update the packet.
+(You should execute the steps regulary to keep the system up to date).
+
+.. code-block :: console
+
+    sudo apt update && sudo apt upgrade
+
+Windows does not update any components of a Linux subsystem, all have to be carried out manually!
+
+Latest Python On Ubuntu
+=======================
+
+Our Linux operating system is installed to author software, so here an explaination how to install the programming language Python.
+To install one or more Python versions on the subsystem, a certain management application for it called Pyenv has to be installed.
+
+Install Pyenv
+-------------
+
+To install Pyenv, use the command below in the Ubuntu shell.
+After completing the download, the shell has to be restarted to take over all changes.
+
+.. code-block :: console
+
+    curl https://pyenv.run | bash
+
+Configure Paths
+---------------
+
+To set the right paths for certain activities and enable Pyenv local environment, edit the file ".bashrc" in Ubuntu using the "nano" editor command.
+
+.. code-block :: console
+
+    nano .bashrc
+
+After opening the document, at the end (bottom of code) 3 lines have to be appended:
+
+.. code-block :: bash
+
+    export PATH="$HOME/.pyenv/bin:$HOME/.local/bin:$PATH"
+    eval "$(pyenv init -)"
+    eval "$(pyenv virtualenv-init -)"
```

Comment:
Compare yours to the [pyenv docs in Step 3](https://github.com/pyenv/pyenv/#basic-github-checkout), modified from a shell command to text that should be appended to the `.bashrc`:

```
export PYENV_ROOT="$HOME/.pyenv"
export PATH="$PYENV_ROOT/bin:$PATH"
if command -v pyenv 1>/dev/null 2>&1; then
 eval "$(pyenv init -)"
fi
```

I have my doubts about using virtualenv because it mutates the $PATH.